### PR TITLE
/observability/what-is-observability: add production profiling to telemetry types

### PR DESCRIPTION
**Pull request by mmanciop**
## Done

Small copy change using one of the [new observability icons](https://github.com/canonical-web-and-design/brand-squad/issues/627#issuecomment-993484902). Changes already reflected in the [copy document](https://docs.google.com/document/d/1eoqDQzFiRJY2u_EO4JBuLBLqae4K3nrEf3qd88KHw0Y/edit#heading=h.extcyact92z9).

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

N/A

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/observability/what-is-observability.html [hunk 1]
```diff
@@ -112,26 +112,40 @@ <h3 class="p-heading--4">Distributed traces</h3>
     <div class="row u-equal-height">
         <div class="col-5 u-align--center u-hide--small u-vertically-center">
             {{ image (
-                url="https://assets.ubuntu.com/v1/f8ac79f6-real-user-monitoring.svg",
-                alt="Real user monitoring",
-                width="180",
-                height="150",
+                url="https://assets.ubuntu.com/v1/32c10814-production-profiling.svg",
+                alt="Production profiling",
```
Comment (anthonydillon) [inline templates/observability/what-is-observability.html:116]: Could you leave the alt text for these icons blank, as it adds no value to the adjacent copy. Only adds replication for screen readers.

File: templates/observability/what-is-observability.html [hunk 2]
```diff
@@ -112,26 +112,40 @@ <h3 class="p-heading--4">Distributed traces</h3>
...
         <div class="col-7">
-            <h3 class="p-heading--4">Synthetic monitoring</h3>
-            <p>Synthetic monitoring checks are about continuously, regularly testing systems, mostly production ones, to ensure that particular APIs or entire user flows in UIs, like a checkout process, work as intended; in other words, synthetic monitoring checks are smoke tests for APIs.</p>
+            <h3 class="p-heading--4">Real user monitoring</h3>
+            <p>Real user monitoring is about collecting data about user sessions occurring via interfaces like Web pages, mobile applications and sometimes IoT devices; real user monitoring is closely related with distributed tracing, in which a user session is correlated with one or more distributed traces, each describing, for example, how one asset or XHR request is served to the user interface.</p>
+        </div>
+        <div class="col-5 u-align--center u-hide--small u-vertically-center">
+            {{ image (
+                url="https://assets.ubuntu.com/v1/f8ac79f6-real-user-monitoring.svg",
+                alt="Real user monitoring",
```
Comment (anthonydillon) [inline templates/observability/what-is-observability.html:138]: Same here

**Review by anthonydillon — CHANGES_REQUESTED** [review]
I have left a comment in the code, regarding the removal of the alt content from the images and have also added some suggestions and comments to the copy document. Could you review the suggestions and incorporate any changes you deem worthy

**Comment by mmanciop** [comment]
@anthonydillon went through the copy, accepted most changes and adjusted others. Thanks for the feedback!

P.S. There were two other minor edits in parts of the page that this PR does not touch that it is worth adopting. I merged them into the copy doc.

**Comment by anthonydillon** [comment]
Thanks @mmanciop. Let me know when you have updated the content in this PR to match the copy doc and I will re-review.

**Comment by mmanciop** [comment]
@anthonydillon second round? :-)

File: templates/observability/what-is-observability.html [hunk 3]
```diff
@@ -63,7 +63,6 @@ <h3 class="p-heading--4">Metrics</h3>
         <div class="col-5 u-align--center u-hide--small u-vertically-center">
             {{ image (
                 url="https://assets.ubuntu.com/v1/df2f25e5-metrics.svg",
-                alt="Metrics",
```
Comment (anthonydillon) [inline templates/observability/what-is-observability.html:66]: You need to leave the alt attribute to be valid mark up but the value should be an empty string.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
The page is 500'ing due to the lack of alt attr on images

File: templates/observability/what-is-observability.html (hunk 3 above)
Comment (mmanciop) [reply templates/observability/what-is-observability.html:66]: Sounds to me like a one-liner fix in the implementation of the tag, but in the interest of getting the update out, sure

**Review by anthonydillon — APPROVED** [review]
LGTM thanks for the updates :+1:

**Pull request merged**